vnet/lib/vnet/node_api/event_base.rb

```diff
@@ -11,6 +11,20 @@ def create(options)
         }
       end
 
+      def update_uuid(uuid, options)
+        model_class[uuid].tap do |model|
+          # TODO: Return error if not found.
+          next if model.nil?
```

Comment:
Figured this is ok for now since there are more pressing matters at hand right now.